scripts/test/release.sh

```diff
@@ -57,18 +57,18 @@ function _test_release_step() {
 }
 
 function test_release() {
-    local version="$1"
-    local status="$2"
-    local expected_fields=("${@:3}")
+    local version=$1
+    local status=$2
+    local branch=$3
```

Comment:
```suggestion
    local version="$1"
    local status="$2"
    local branch="$3"
```

Reply:
Please explain why you think this is needed?

In bash it's not required, and if we're concerned about porting scripts to other shells this is just part of the tings that might not work (though shells other than bash support this usage as well) -> https://unix.stackexchange.com/questions/97560/are-quotes-needed-for-local-variable-assignment

Reply:
It’s not needed, but I think it’s simpler for maintenance purposes to have quotes everywhere. Otherwise, we’re expecting people looking at this to know what these assignments are fine, and more generally to know in detail when quotes are needed and when they’re not. See the last paragraph in the accepted answer on the question above ;-).